chartfx-chart/src/main/java/io/fair_acc/chartfx/axes/spi/AbstractAxis.java

```diff
@@ -598,6 +598,43 @@ private void applyOverlapPolicy(List<TickMark> tickMarks) {
 
     }
 
+    /**
+     * Hides labels that can't be fully displayed due to the
+     * bounds of the parent container. This often manifests
+     * on the top of y axes when there is no title padding.
+     * <p>
+     * Note that the layout bounds may not be available
+     * until the bounds phase, so we use layout x/y directly.
+     */
```

Comment:
If people want to show e.g. the rightmost tick on their x axis without having something to show on the right side of the chart, I assume they could just add some padding to the right side of the chart?

Reply:
yes, they could add padding via CSS

```
.chart-content {
  -fx-padding: 5px;
}
```